src/domain/app/header/Header.tsx

```diff
@@ -1,15 +1,108 @@
-import React from 'react';
+import { Navigation } from 'hds-react';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory, useLocation } from 'react-router-dom';
 
+import { MAIN_CONTENT_ID, SUPPORT_LANGUAGES } from '../../../constants';
+import useLocale from '../../../hooks/useLocale';
+import { OptionType } from '../../../types';
+import { updateLocaleParam } from '../../../util/updateLocaleParam';
+import { ROUTES } from '../constants';
 import styles from './header.module.scss';
-import MobileNavigation from './mobileNavigation/MobileNavigation';
-import Navbar from './navbar/Navbar';
 
-const Header: React.FC = () => {
+export interface HeaderProps {
+  menuOpen: boolean;
+  onMenuToggle: () => void;
+}
+const Header: React.FC<HeaderProps> = ({ menuOpen, onMenuToggle }) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const location = useLocation();
+  const locale = useLocale();
+
+  const languageOptions: OptionType[] = React.useMemo(() => {
+    return Object.values(SUPPORT_LANGUAGES).map((language) => ({
+      label: t(`header.languages.${language}`),
+      value: language,
+    }));
+  }, [t]);
+
+  const changeLanguage = (newLanguage: OptionType) => (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+  ) => {
+    event.preventDefault();
+    history.push({
+      pathname: updateLocaleParam(location.pathname, locale, newLanguage.value),
+      search: location.search,
+    });
+  };
+
+  const isTabActive = (pathname: string): boolean => {
+    return location.pathname.startsWith(pathname);
+  };
+
+  const goToPage = (pathname: string) => (
+    event?: React.MouseEvent<HTMLAnchorElement>
+  ) => {
+    event?.preventDefault();
+    history.push({ pathname });
+  };
+
+  const logoLang = locale === 'sv' ? 'sv' : 'fi';
+
+  const navigationItems = [
+    {
+      label: t('header.searchEvents'),
+      url: `/${locale}${ROUTES.EVENTS}`,
+    },
+    {
+      label: t('header.searchCollections'),
+      url: `/${locale}${ROUTES.COLLECTIONS}`,
+    },
+  ];
   return (
-    <header className={styles.headerWrapper}>
-      <Navbar />
-      <MobileNavigation />
-    </header>
+    <Navigation
+      menuOpen={menuOpen}
+      onMenuToggle={onMenuToggle}
+      menuToggleAriaLabel={t('header.menuToggleAriaLabel')}
+      skipTo={`${location.pathname}${location.search}#${MAIN_CONTENT_ID}`}
```

Comment:
It's not related to this PR. But it's little bit problematic that skipTo prop is just href of an anchor. I haven't tested this PR but I guess that after clicking "skip to content" link the page is totally reloaded and the Apollo cache is cleared. 

It would be nice to have onSkipTo where you could use react-router and prevent anchor event (same way as onTitleClick)

But as said not related to this PR.

Reply:
I tested that now. Seems to work correctly, it doesn't refresh the page. Maybe it has something to do with the hash, because without it, it refreshes

Reply:
https://stackoverflow.com/questions/4855168/what-is-href-and-why-is-it-used#:~:text=A%20hash%20%2D%20%23%20within%20a%20hyperlink,the%20id%20on%20that%20page.

explaned there. Hyperlink with hash only scrolls

Reply:
BUT. I think i just figured that it only scrolls because the current url is same but you could also just pass the hash:

`skipTo={`#${MAIN_CONTENT_ID}`}`

Reply:
Good, that it works. I guess you meant
```
skipTo={`#${MAIN_CONTENT_ID}`}
```

Reply:
yeah I tried my best to escape those ticks but couldn't !!!! 🤯

Reply:
done